assets/src/edit-story/components/button/icons/eyedropper.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Height, width and fills should be removed in svgs.

Reply:
Thanks to #244 this is done automatically during build 🙂 

`fill="none"` is fine btw, just the `fill="white"` below would be problematic if we wouldn't have our new config.